config/services.yaml

```diff
@@ -30,6 +30,7 @@ services:
         arguments:
             - '%doctrine.default_connection%'
             - '%db_tools.excluded_tables%'
+            - '%db_tools.backupper.options%'
```

Comment:
Considering one might want to use Backupper alone, I don't know if this should'nt be an argument of the `db_tools.backupper.factory` service.

But considering we already give `%db_tools.excluded_tables%` this way, it seems fair to do it like you did.

Reply:
> But considering we already give %db_tools.excluded_tables% this way, it seems fair to do it like you did.

That's why I did it this way :wink:.